lib/utils/parseRequest.js

```diff
@@ -1,3 +1,14 @@
 module.exports = rawRequest => {
+  const parsedRequest = rawRequest.split('\r\n');
+  const [method, path] = parsedRequest[0].split(' ');
+  if(parsedRequest[7]){
+    const body = parsedRequest[7];
+    return ({ method, path, body });
+  }
+  if(parsedRequest[2]){
+    const body = parsedRequest[2];
+    return ({ method, path, body });
+  }
+  return ({ method, path });
```

Comment:
Note that you don't need the parenthesis on any of these return statements:

```suggestion
  return { method, path };
```